README.md

```diff
@@ -431,6 +432,7 @@ A curated list of awesome Java frameworks, libraries and software.
 - [Retrofit](https://square.github.io/retrofit) - Typesafe REST client.
 - [Ribbon](https://github.com/Netflix/ribbon) - Client-side IPC library that is battle-tested in cloud.
 - [Riptide](https://github.com/zalando/riptide) - Client-side response routing for Spring's RestTemplate.
+- [Scarlet](https://github.com/Tinder/Scarlet) - A Retrofit inspired WebSocket client for Kotlin, Java, and Android.
```

Comment:
Not sure about this, it's written in Kotlin and examples are in Kotlin. It's a general issue for the list since JVM languages are compatible.

Reply:
They are compatible but often not idiomatic and very odd to use in Java.